test/nn/encoder/test_stype_encoder.py

```diff
@@ -148,6 +149,35 @@ def test_multicategorical_feature_encoder(encoder_cls_kwargs):
     assert (x_perturbed[:, 1:, :] == x[:, 1:, :]).all()
 
 
+@pytest.mark.parametrize(
+    "encoder_cls_kwargs",
+    [(
+        Time2VecEncoder,
+        {
+            "na_strategy": NAStrategy.ZEROS
+        },
+    )],
+)
+def test_timestamp_feature_encoder(encoder_cls_kwargs):
+    dataset: Dataset = FakeDataset(num_rows=10, stypes=[stype.timestamp])
+    dataset.materialize()
+    tensor_frame = dataset.tensor_frame
+    stats_list = [
+        dataset.col_stats[col_name]
+        for col_name in tensor_frame.col_names_dict[stype.timestamp]
+    ]
+
+    encoder = encoder_cls_kwargs[0](
+        8,
+        stats_list=stats_list,
+        stype=stype.timestamp,
+        **encoder_cls_kwargs[1],
+    )
+    feat_timestamp = tensor_frame.feat_dict[stype.timestamp]
+    x = encoder(feat_timestamp)
+    assert x.shape == (feat_timestamp.size(0), feat_timestamp.size(1), 8)
```

Comment:
Can we add a final integration test on `test_stypewise_encoder.py`?

Reply:
I need to figure out a good na strategy for timestamps first. Let’s discuss offline tomorrow!